tests/libvmi/factory.go

```diff
@@ -89,3 +89,16 @@ func NewCirros(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	cirrosOpts = append(cirrosOpts, opts...)
 	return New(RandName(DefaultVmiName), cirrosOpts...)
 }
+
+// NewAlpine instantiates a new Alpine based VMI configuration
+func NewAlpine(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	alpineOpts := []Option{
+		WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
+		WithCloudInitNoCloudUserData("#!/bin/bash\necho 'hello'\n", true),
+		WithResourceMemory("128Mi"),
+		WithRng(),
+		WithTerminationGracePeriod(DefaultTestGracePeriod),
+	}
+	alpineOpts = append(alpineOpts, opts...)
+	return New(RandName(DefaultVmiName), alpineOpts...)
```

Comment:
nit: I would inline these.

You're welcome to prefer your alternative.

Reply:
You mean `return New(RandName(DefaultVmiName), append(alpineOpts, opts...)...)` ?
It looks loaded and hard to read.

Reply:
> You mean `return New(RandName(DefaultVmiName), append(alpineOpts, opts...)...)` ? It looks loaded and hard to read.

```golang
return New(
    RandName(DefaultVmiName), 
    append(alpineOpts, opts...)...)
```
reads very well IMO. Pretty obvious the first parameter is a name, and the second are a slice of alpine options. Whatever those are.

Again, you're welcome to keep it as is.